org/models.py

```diff
@@ -1,10 +1,57 @@
 from django.db import models
+from django.utils import timezone
+from .custom_model_field import *
+from users.models import User
 
 # Create your models here.
-from .custom_model_field import *
 
-class PermissionSetTestModel(models.Model):
-    perm = PermissionField()
```

Comment:
Please dont remove this model as any future changes in the PermissionField model field is needed to be tested. And the testing of migrations is done with the help of this model.